### ISSUE TITLE
BroadcastChannelIsPartitioned test is flaky

### ISSUE THREAD

**Issue by goodov** [brave/brave-browser#20130]
```
[  FAILED  ] EphemeralStorageBrowserTest.BroadcastChannelIsPartitioned
Stacktrace
Failure at ../../brave/browser/ephemeral_storage/ephemeral_storage_browsertest.cc (line 895):
Expected equality of these values:
  kTestMessage
    Which is: 0x11de483a1
  GetBroadcastMessage(rfh)
    Which is: ""
```

**Comment by kjozwiak** [comment]
Ran into this as well while looking at https://github.com/brave/brave-core/pull/11658 for uplift.

```
08:40:50  [ RUN      ] EphemeralStorageBrowserTest.BroadcastChannelIsPartitioned
08:40:50  [70485:259:1221/083943.043846:INFO:content_main_runner_impl.cc(1133)] Chrome is running in full browser mode.
08:40:50  [70485:259:1221/083943.290111:WARNING:external_provider_impl.cc(502)] Malformed extension dictionary for extension: odbfpeeihdkbihmopkbjmoonfanlbfcl.  Key external_update_url has value "", which is not a valid URL.
08:40:50  [70485:259:1221/083943.552789:ERROR:native_widget_mac.mm(410)] Not implemented reached in virtual const gfx::ImageSkia *views::NativeWidgetMac::GetWindowIcon()
08:40:50  [70485:259:1221/083943.563254:ERROR:native_widget_mac.mm(414)] Not implemented reached in virtual const gfx::ImageSkia *views::NativeWidgetMac::GetWindowAppIcon()
08:40:50  [70485:259:1221/083944.275652:WARNING:CONSOLE(1)] "no tab id so cannot request shield data from window focus change!", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/out/brave_extension_background.bundle.js (1)
08:40:50  [70485:259:1221/083944.324231:ERROR:CONSOLE(1)] "[Shields]: Can't request shields panel data for tabId: 3.  Error: No tab url specified", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/out/brave_extension_background.bundle.js (1)
08:40:50  [70493:7171:1221/083944.592848:WARNING:simple_synchronous_entry.cc(1326)] Could not open platform files for entry.
08:40:50  [70485:37123:1221/083944.655423:ERROR:chrome_browser_main_extra_parts_metrics.cc(227)] START: ReportBluetoothAvailability(). If you don't see the END: message, this is crbug.com/1216328.
08:40:50  [70485:37123:1221/083944.655647:ERROR:chrome_browser_main_extra_parts_metrics.cc(230)] END: ReportBluetoothAvailability()
08:40:50  [70485:37123:1221/083944.655729:ERROR:chrome_browser_main_extra_parts_metrics.cc(235)] START: GetDefaultBrowser(). If you don't see the END: message, this is crbug.com/1216328.
08:40:50  [70485:37123:1221/083944.656504:ERROR:chrome_browser_main_extra_parts_metrics.cc(239)] END: GetDefaultBrowser()
08:40:50  [70485:259:1221/083945.402500:ERROR:CONSOLE(1)] "[Shields]: Can't request shields panel data for tabId: 4.  Error: No tab url specified", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/out/brave_extension_background.bundle.js (1)
08:40:50  [70485:259:1221/083945.402826:ERROR:CONSOLE(1)] "[Shields]: Can't request shields panel data for tabId: 4.  Error: No tab url specified", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/out/brave_extension_background.bundle.js (1)
08:40:50  [70485:259:1221/083945.403065:ERROR:CONSOLE(1)] "[Shields]: Can't request shields panel data for tabId: 4.  Error: No tab url specified", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/out/brave_extension_background.bundle.js (1)
08:40:50  [70485:259:1221/083946.064259:ERROR:CONSOLE(1)] "[Shields]: Can't request shields panel data for tabId: 5.  Error: No tab url specified", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/out/brave_extension_background.bundle.js (1)
08:40:50  [70485:259:1221/083946.065112:ERROR:CONSOLE(1)] "[Shields]: Can't request shields panel data for tabId: 5.  Error: No tab url specified", source: chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/out/brave_extension_background.bundle.js (1)
08:40:50  [70485:259:1221/083946.368867:WARNING:brave_stats_updater_params.cc(129)] Couldn't find the time of first run. This should only happen when running tests, but never in production code.
08:40:50  ../../brave/browser/ephemeral_storage/ephemeral_storage_browsertest.cc:895: Failure
08:40:50  Expected equality of these values:
08:40:50    kTestMessage
08:40:50      Which is: 0x118d692e1
08:40:50    GetBroadcastMessage(rfh)
08:40:50      Which is: ""
08:40:50  
08:40:50  Stack trace:
08:40:50  0   brave_browser_tests                 0x000000010b05d838 brave_browser_tests + 3426360
08:40:50  1   brave_browser_tests                 0x0000000115172346 brave_browser_tests + 172331846
08:40:50  2   brave_browser_tests                 0x000000010b201950 brave_browser_tests + 5146960
08:40:50  3   brave_browser_tests                 0x00000001106720fe brave_browser_tests + 93688062
08:40:50  4   brave_browser_tests                 0x00000001106714e8 brave_browser_tests + 93684968
08:40:50  5   brave_browser_tests                 0x000000010cf1402f brave_browser_tests + 35631151
08:40:50  6   brave_browser_tests                 0x000000010d5f53e6 brave_browser_tests + 42845158
08:40:50  7   brave_browser_tests                 0x000000010cf13c8f brave_browser_tests + 35630223
08:40:50  8   brave_browser_tests                 0x000000010cf1640b brave_browser_tests + 35640331
08:40:50  9   brave_browser_tests                 0x000000010cf123d6 brave_browser_tests + 35623894
08:40:50  10  brave_browser_tests                 0x000000010e72cb81 brave_browser_tests + 60898177
08:40:50  11  brave_browser_tests                 0x000000010e72c545 brave_browser_tests + 60896581
08:40:50  12  brave_browser_tests                 0x000000010e72add0 brave_browser_tests + 60890576
08:40:50  13  brave_browser_tests                 0x000000010e72b6fe brave_browser_tests + 60892926
08:40:50  14  brave_browser_tests                 0x0000000115171396 brave_browser_tests + 172327830
08:40:50  15  brave_browser_tests                 0x00000001114b5973 brave_browser_tests + 108644723
08:40:50  
08:40:50  [70485:259:1221/083951.653935:ERROR:node_channel.cc(858)] Dropping message on closed channel.
08:40:50  [70485:48131:1221/083951.660013:ERROR:process_posix.cc(330)] Unable to terminate process 70493: No such process (3)
08:40:50  [  FAILED  ] EphemeralStorageBrowserTest.BroadcastChannelIsPartitioned, where TypeParam =  and GetParam() =  (8794 ms)
```

**Comment by goodov** [comment]
Need to apply another fix, will uplift to 1.35.x when will be merged.
